src/sql/workbench/api/common/extHostModelViewDialog.ts

```diff
@@ -457,6 +457,27 @@ class WizardImpl implements azdata.window.Wizard {
 	}
 }
 
+class ModelViewDashboardImpl implements azdata.window.ModelViewDashboard {
+	constructor(
+		private _editor: ModelViewEditorImpl
+	) {
+	}
+
+	registerTabs(handler: (view: azdata.ModelView) => Thenable<(azdata.Tab | azdata.TabGroup)[]>): void {
```

Comment:
Just to be clear - this will allow modifying the tabs after creation of the dashboard? That's going to be something we want/need for the Arc/BDC dashboards if not

Reply:
And will we be able to easily remove tabs as well? Will removing them from the container handle it correctly?

Reply:
good, lets talk about these requirement in tomorrow's meeting.

Reply:
will implement add remove tab in a separate PR